charts/jenkins/pod-templates/nuxeo-web-ui.yaml.gotmpl

```diff
@@ -0,0 +1,32 @@
+- id: "nuxeo-web-ui"
+  name: "nuxeo-web-ui"
+  label: "jenkins-nuxeo-web-ui"
+  imagePullSecrets:
+  - name: {{ .Values.podTemplates.imagePullSecretName }}
+  serviceAccount: {{ .Values.podTemplates.serviceAccount }}
+  nodeUsageMode:  {{ .Values.podTemplates.nodeUsageMode }}
+  containers:
+  {{- toYaml .Values.podTemplates.containers | nindent 2 -}}
+  - args: ""
+    command: ""
+    image: "{{ .Values.docker.registry }}/nuxeo/builder-chrome-nodejs16:latest"
+    alwaysPullImage: true
+    name: "ftests"
+    privileged: {{ .Values.podTemplates.privileged }}
+    resourceLimitCpu: "3"
+    resourceLimitMemory: "6Gi"
+    resourceRequestCpu: "1.5"
+    resourceRequestMemory: "4Gi"
+    ttyEnabled: {{ .Values.podTemplates.ttyEnabled }}
+    workingDir: {{ .Values.podTemplates.workingDir }}
+  envVars:
+  {{- toYaml .Values.podTemplates.envVars | nindent 2 -}}
+  {{- toYaml .Values.podTemplates.javaEnvVars | nindent 2 -}}
+  {{- toYaml .Values.podTemplates.mavenEnvVars | nindent 2 -}}
```

Comment:
These 2 aren't needed since they're for executing Java or Maven, which won't be the case in the ftests container.